e3sm_diags/driver/lat_lon_driver.py

```diff
@@ -4,55 +4,110 @@
 import os
 
 import cdms2
-import MV2
 
 import e3sm_diags
 from e3sm_diags.driver import utils
 from e3sm_diags.logger import custom_logger
-from e3sm_diags.metrics import corr, max_cdms, mean, min_cdms, rmse, std
+from e3sm_diags.metrics import corr, mean, rmse, std
```

Comment:
max_cdms and min_cdms no longer need to be imported. They are still being used in other drivers, we can consider remove all instances in a future house cleaning PR.